AK/String.cpp

```diff
@@ -196,4 +196,48 @@ String String::repeated(char ch, int count)
     return *impl;
 }
 
+// TODO: case insensitive matching. consider adding a toupper/tolower implementation in String?
```

Comment:
String has to_uppercase() and to_lowercase(), is that what you had in mind?

Reply:
Hmm. I could have sworn I didn't see that :)

I can throw it in as an enum then already.